### ISSUE TITLE
I ran the code successfully but can't receive the answer 

### ISSUE THREAD

**Issue by Jane1702** [PromtEngineer/localGPT#518]
![Screenshot 2023-09-22 162841](https://github.com/PromtEngineer/localGPT/assets/94185643/d63f67a1-ea9f-4d0d-bfe3-2c006e6ecad3)

**Comment by RobertoDonPedro** [comment]
I experience the same issue - No response,
My setup:
- LInux Debian 11 (bullseye)
- 2 Nvidia T4 GPUs with 16GB GPU memory each
- cuBLAS enabled with `CMAKE_ARGS="-DLLAMA_CUBLAS=on" FORCE_CMAKE=1 pip install llama-cpp-python==0.1.83 --no-cache-dir`

<img width="795" alt="image" src="https://github.com/PromtEngineer/localGPT/assets/19687084/94f1078b-534e-409e-85fb-474d510290de">

**Comment by faiconda** [comment]
I have the same problem

**Comment by PromtEngineer** [comment]
@Jane1702 @RobertoDonPedro @faiconda are you using the default model?

**Comment by RobertoDonPedro** [comment]
> @Jane1702 @RobertoDonPedro @faiconda are you using the default model?

Yes, but I tried it with several other models as well, including the GPTQ models since I have 2 GPUs. But it just does not answer.

**Comment by Jane1702** [comment]
> @Jane1702 @RobertoDonPedro @faiconda are you using the default model?

Yes , I used the default model

**Comment by RobertoDonPedro** [comment]
I just found the issue in my case. It was not the model, but this line in the run_localGPT.py file:
`query = input("\nEnter a query: ")`
It does not read the provided input and waits forever. If I replace this line manually in the code with e.g. `query = "Summarize your content" `everything works as expected.
I think this is caused by running localGPT with VSCode on a remote VM. However, still wondering what's the issue with VSCode in this regard.

**Comment by RobertoDonPedro** [comment]
> Excuse-me , when I changed the code as you said , it finished the code but it didn't return the answer ![Screenshot 2023-09-25 170015](https://user-images.githubusercontent.com/94185643/270384924-ef9688ff-269d-4ad8-bd97-484859558f45.png)

..then the root of your issue is something else than in my case. 
You did ingest the data, right?

**Comment by RobertoDonPedro** [comment]
Just solved my issue. It wasn't related to VS Code but to the fact that I called it from a nodejs app via shelljs and shelljs does not accept input. Now I call it natively via child_process.execFileSync with the stdio: 'inherit' parameter and now it works as expected. 
So in summary my issue wasn't neither related to localGPT nor to VS Code but to they way I called localGPT. Potentially @Jane1702 you could also doublecheck the way you call it.

**Comment by Jane1702** [comment]
> Just solved my issue. It wasn't related to VS Code but to the fact that I called it from a nodejs app via shelljs and shelljs does not accept input. Now I call it natively via child_process.execFileSync with the stdio: 'inherit' parameter and now it works as expected. So in summary my issue wasn't neither related to localGPT nor to VS Code but to they way I called localGPT. Potentially @Jane1702 you could also doublecheck the way you call it.

Sorry , I don't understand how to do it , can you give me an example please ?

**Comment by Jane1702** [comment]
![Screenshot 2023-09-26 143337](https://github.com/PromtEngineer/localGPT/assets/94185643/515de56b-ed20-4b86-8906-9b2605ca25a8)
 I don't know why but today I tried again and it worked normally . But it takes so long to have 1 answer (more than 20mins)

**Comment by KansaiTraining** [comment]
It took 20 minutes?!

(I am having the same problem)
